src/cpu.cpp

```diff
@@ -16,15 +16,35 @@
 
 /* rdtsc for MSC_VER, uses simple inline assembly, or _rdtsc
  * from external win64.asm because VS2005 does not support inline assembly */
-#if defined(_MSC_VER) && (defined(_M_IX86) || defined(_M_X64)) && !defined(RDTSC_AVAILABLE)
+#if defined(_MSC_VER) && !defined(RDTSC_AVAILABLE)
 #include <intrin.h>
+#include <windows.h>
 uint64 ottd_rdtsc()
 {
+#if defined(_M_ARM)
+	return __rdpmccntr64();
```

Comment:
just to ask the question: do we want to add code for arm32, while we don't support it?

Reply:
I don't see that it hurts to, people could choose to build it manually. We could enable ARM32 support, I just don't think there's a lot of point. :)